pkg/crc/preflight/preflight_checks_linux.go

```diff
@@ -4,6 +4,7 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
+	"golang.org/x/sys/unix"
```

Comment:
should be defined below the golang framework imports, but before our own. right?